projects/examples/standalone/src/app/services/book.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { Book } from '../models/book.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BookService {
+  private apiUrl = 'http://localhost:8181/api/books';
+
+  constructor(private http: HttpClient) {}
```

Comment:
you could use injections instead of importing the HttpClient inside the constructor: 
`import { inject } from '@angular/core';`

 `private http = inject(HttpClient)`

Reply:
Changed. 
Thank you @mrs-cp 🙂